pkg/synopsysctl/cmd_deploy.go

```diff
@@ -331,6 +332,10 @@ var deployCmd = &cobra.Command{
 			return err
 		}
 
+		// In some rare cases, the CRD may not be available when we create the default sizes.
+		// Adding a 3 seconds delay as a workaround
+		time.Sleep(3 * time.Second)
```

Comment:
Let's not do a hardcoded sleep.  Prefer to do a get and wait for the CRD to exist with a timeout rather than a sleep.

Reply:
Yash and I wrote some functions here that we might be able to use: https://github.com/blackducksoftware/cloud-native-tests/blob/master/utils/k8shelper/crd/crdutils.go

Reply:
Also see api-machinery's wait.PollImmediate: https://github.com/kubernetes/apimachinery/blob/master/pkg/util/wait/wait.go#L325

Reply:
Updated